ci-operator/jobs/openshift/cluster-api-provider-alibaba/openshift-cluster-api-provider-alibaba-main-presubmits.yaml

```diff
@@ -71,6 +71,7 @@ presubmits:
         - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
         - --report-credentials-file=/etc/report/credentials
         - --target=[images]
+        - --target=[release:latest]
```

Comment:
This seems to be causing an issue with the build, was this generated?

Reply:
yes, it was generated